specification/ai/Azure.AI.Projects/datasets/routes.tsp

```diff
@@ -30,6 +31,7 @@ interface Datasets
     {
       @doc("The pending upload request parameters")
       @Http.bodyRoot
+      @clientName("configuration")
```

Comment:
This should go in client.tsp unless there is some obscure reason it can't?

Reply:
@johanste as far as I know that's not possible to do (declare this rename in client.tsp), unless you define a template where the client name is a template argument. I think that will obfuscate the TypeSpec too much, just for the sake of overwriting client name. @iscai-msft @lmazuel what are your thoughts?